src/meta.rst

```diff
@@ -73,10 +73,23 @@ A full description of selectors is
 
 Building Against NumPy
 ----------------------
-If you have a package which links against numpy you need to build and run against
-the same version of numpy. Putting ``numpy x.x`` in the build and run requirements
-ensures that a separate package will be built for each version of numpy that
-conda-forge builds against.
+If you have a package which links\* against numpy you need to build against the oldest possible version of numpy that is forwards compatible.
+That can be achieved by pinning the build requirements and letting "free" the run requirements.
+At the moment these are the oldest available numpy versions in conda-forge that you can use:
```

Comment:
These are not the oldest available versions in conda-forge right? They are pulled in from defaults, right?